### PR TITLE
DEV-3505-feat(piscine-go): add README.md for new go exercise rockandroll

### PR DESCRIPTION
**Pull request by tiagocollot**
Created README.md for the new go exercise rockandroll

### CODE REVIEW

File: subjects/rockandroll/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## rockandroll
+
+### Instructions
+
+Write a function called `RockAndRoll` that takes an `int` and returns a `string`.
+
+- If the number is divisible by 2, print `rock` followed by a newline `\n`.
+- If the number is divisible by 3, print `roll` followed by a newline `\n`.
+- If the number is divisible by 2 and 3, print `rock and roll` followed by a newline `\n`.
+
```
Comment (jotapero) [inline subjects/rockandroll/README.md:10]: Missing two edge cases (maybe three depending on whether we should test for overflows or not). What happens with negative numbers, and numbers that are not divisible by 2 or 3). Function prints the number on these two cases

File: subjects/rockandroll/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import (
+	"fmt"
+	"strconv"
+)
+
+func main() {
+	fmt.Println(RockAndRoll(4))
+	fmt.Println(RockAndRoll(9))
+	fmt.Println(RockAndRoll(6))
```
Comment (jotapero) [inline subjects/rockandroll/README.md:33]: Maybe add here edge cases or simply mentioning them above is enough. For instance:
fmt.Println(RockAndRoll(-10)) = ? what should it print
fmt.Println(RockAndRoll(11)) = ?

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/rockandroll/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/rockandroll/README.md:10]: I think it is enough that the test already covers these cases There is no need to be present in the subject.

File: subjects/rockandroll/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/rockandroll/README.md:33]: Same logic as the above answer!

Comment (tiagocollot) [reply subjects/rockandroll/README.md:33]: Same logic as the answer above.

File: subjects/rockandroll/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/rockandroll/README.md:10]: I think it is enough, the test already covers these cases. There is no need to be present in the subject.

**Review by jotapero — APPROVED** [review]

File: subjects/rockandroll/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+func main() {
+  fmt.Println(piscine.RockAndRoll(4))
+  fmt.Println(piscine.RockAndRoll(9))
+  fmt.Println(piscine.RockAndRoll(6))
+}
+```
+And its output:
+
+```go
```
[the remark this answers is not in the document]
Comment (tiagocollot) [reply subjects/rockandroll/README.md]: Yes it should, it is fixed! Thank you.

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Review by eslopfer — APPROVED** [review]
